contrib/cmake/preferred-ports-megacmd.txt

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Would it make sense for these to go into MEGAcmd/MEGAsync/... projects instead of here ... :thinking:

Reply:
yes, but for ease of development and review I've left them here for now - if and when this new way to build 3rdParty is approved, let's move them to the appropriate repos.  thanks